typescript/src/slimselect.ts

```diff
@@ -0,0 +1,19 @@
+import SlimSelect from "slim-select";
+
+
+const slimSelectClass = "use-slimselect";
+
+
+
+setupSlimSelect();
+
+
+
+//
+// Transform <select> with SlimSelect
+//
+function setupSlimSelect() {
+  Array.from(document.querySelectorAll("." + slimSelectClass)).forEach(function(select) {
+    new SlimSelect({ select: select, selectByGroup: true, closeOnSelect: false });
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/betagouv/aplus/pullRequest?prid=7859967)
```suggestion
    new SlimSelect({ select, selectByGroup: true, closeOnSelect: false });
```